### DISCUSSION TITLE
is the 4.1 web UI refresh really going to look like that? (color scheme)

### DISCUSSION THREAD

**Discussion by ghost** [netbox-community/netbox#17091] [Q&A]
https://beta-demo.netbox.dev/

So I wasn't a huge fan of the dark menu interface on light theme: https://github.com/netbox-community/netbox/issues/16876
But the 4.1 beta site has this new green/teal tinge to the fonts and a gradient background on the nav menu.

Is this permanent? While I thought there are certainly ways to improve the 4.0 UI, the 4.1 changes are not my cup of tea. Are the only options going to be either 100% dark mode, or this new teal tinge 100% light mode?
Is the Netbox blue nav bar gone for good?

**Comment by candlerb** [comment]
I prefer the new light navigation to 4.0, but the problem I see is that the new font is fatter, especially the zeros. This means more horizontal space consumed. It's not a huge difference, but I noticed it immediately.

Example comparing demo.netbox.dev with beta-demo.netbox.dev:

![image](https://github.com/user-attachments/assets/21c3bf40-931e-41d9-bc44-45de50e962a7)

In the new font a number 0 is as fat as I'd expect letter O, and letter O is almost a full circle:

<img width="219" alt="image" src="https://github.com/user-attachments/assets/974ecf27-f71b-437d-9e07-08a3a51f5f41">

**Reply by ghost** [reply]
> I prefer the new light navigation to 4.0,

I mean, I prefer the new light sidebar too, just not the green/teal font color choices.

If I could **choose**, I'd choose the existing 4.0 light theme. Will users get a choice?

**Reply by seismiccollision** [reply]
i also like the new light theme but in dark theme the teal link color is coming across as too neon for my taste

it's also bugged me since (i think) version 4.0 that the table line edit button is "btn-warning" colored but the page item edit button is "btn-yellow" i'd vote both be yellow personally but anything that keeps the function a consistent color would be nice

**Comment by dreng** [comment]
Personally I don't like the green theme at all. The blue color theme looked pretty and subtle, quite the opposite of the green theme.

While this is only a subjective opinion, there is a much more serious problem with the green font: in dark mode it is very tiring and difficult to read. The same applies to the grayish/white font. Please don't leave it as it is.

**Comment by PieterL75** [comment]
I'm not a fan of the font either. looks too 'round'..
And the menu in the light mode is pretty pale. You don't see the difference between the main and subitems
![image](https://github.com/user-attachments/assets/c7107b51-88c8-452b-98fc-f9a67ad2a7e7)


and the dark mode.. no need to turn on the light anymore !
![image](https://github.com/user-attachments/assets/c2e9ecd7-9cd2-4ba2-bb41-4fd83dc3b627)

**Comment by PaulR282** [comment]
Since I also didn't like the color if the UI refresh, I wrote a little plugin to recolor netbox, so the text is blue again. If anyone is interested: https://github.com/PaulR282/netbox_recolor

**Reply by ghost** [reply]
interesting. can you add some screenshots to your git to give someone an idea of the changes?
were you able to do anything about the left menu navigation bar?

**Reply by PaulR282** [reply]
Hi, I have just added example images.

**Reply by rusomenace** [reply]
> Since I also didn't like the color if the UI refresh, I wrote a little plugin to recolor netbox, so the text is blue again. If anyone is interested: https://github.com/PaulR282/netbox_recolor

Hey Paul nice colors, sorry to Task but how do you install it? Thanks

**Reply by pbrinkhoff** [reply]
I'm also interested in how to install the plugin. @PaulR282

**Reply by PaulR282** [reply]
Hi, sorry for the late replay @rusomenace @pbrinkhoff 

You can just install it like every other plugin. If you use Docker, you can just copy the plugin in the container and run `/opt/netbox/venv/bin/pip install --editable /path/recolor` in the Docker file. Then you just need to activate it in the `configuration.py`. Let me know if you need further assistance

**Reply by pbrinkhoff** [reply]
@PaulR282 I've installed NetBox from as release archive.
https://netboxlabs.com/docs/netbox/en/stable/installation/3-netbox/ (option: A)

**Reply by candlerb** [reply]
How to install plugins is documented here: https://netboxlabs.com/docs/netbox/en/stable/plugins/installation/

**Reply by pbrinkhoff** [reply]
@PaulR282 
I used: 
```
pip install netbox_recolor/
Processing ./netbox_recolor
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: netbox-recolor
  Building wheel for netbox-recolor (setup.py) ... done
  Created wheel for netbox-recolor: filename=netbox_recolor-1.0-py3-none-any.whl size=1683 sha256=9badb1e75f62155949831f9701ae1aaa553f5e5b1b2a55b86c4202523c70f187
  Stored in directory: /tmp/pip-ephem-wheel-cache-ougswght/wheels/fa/a5/3f/fbcbafb547ce5522a06f480b8afbba1603d71f793fc3f8bede
Successfully built netbox-recolor
Installing collected packages: netbox-recolor
Successfully installed netbox-recolor-1.0
```
And edited configuration.py:
`PLUGINS = ["other-plugin","netbox-recolor"]`
And restarted netbox netbox-rq
But now `journalctl -eu netbox` shows:
django.core.exceptions.ImproperlyConfigured: Unable to import plugin netbox-recolor: Module not found. Check that the plugin module has been installed within the correct
Any idea?

**Reply by PaulR282** [reply]
@pbrinkhoff You need to add it with `netbox_recolor` and not `netbox-recolor`. I will write an installation instruction later.

**Comment by pheeef** [comment]
I'm happy to see some design overhauls but this teal colour scheme breaks consistency that was previously present with the blue design. 

For instance the table background colour is still a 'dark-blue'/slate ~~like colour. In my opinion it visually clashes with the teal text.

This is complaining on a high level but I feel like it should be said: I don't like how the new logo looks, both because of the colour and the letter spacing seems off. And switching between a dark grey and a white along with having colour is not good for providing a consistent visual experience 

![20240923_13h49m11s_grim](https://github.com/user-attachments/assets/e68ac265-cc9d-4ee3-b66a-7a62202f71b2)
![20240923_13h49m17s_grim](https://github.com/user-attachments/assets/26058fca-69f6-4762-8596-95dc51e22fe4)

I myself consider the old logo to be better than the new design.

**Reply by PaulR282** [reply]
In the plugin I mentioned in the comment earlier, there is a description of how to change the logo to the old one.

**Reply by pheeef** [reply]
Thank you, I'll take a look :)

**Comment by mikygee** [comment]
Hello,
Does anyone know which font has been used for the Netbox logo ? Word Netbox and/or Community.
